rds.tf

```diff
@@ -0,0 +1,127 @@
+resource "aws_db_subnet_group" "rds" {
+  name = "internal_tableau_rds_group"
+
+  subnet_ids = [
+    "${aws_subnet.subnet.id}",
+    "${aws_subnet.internal_tableau_az2.id}",
+  ]
+
+  tags {
+    Name = "rds-subnet-group-${local.naming_suffix}"
+  }
+}
+
+resource "aws_subnet" "internal_tableau_az2" {
+  vpc_id                  = "${var.apps_vpc_id}"
+  cidr_block              = "${var.dq_internal_dashboard_subnet_cidr_az2}"
+  map_public_ip_on_launch = false
+  availability_zone       = "${var.az2}"
+
+  tags {
+    Name = "az2-subnet-${local.naming_suffix}"
+  }
+}
+
+resource "aws_route_table_association" "internal_tableau_rt_rds" {
+  subnet_id      = "${aws_subnet.internal_tableau_az2.id}"
+  route_table_id = "${var.route_table_id}"
+}
+
+resource "random_string" "password" {
+  length  = 16
+  special = false
+}
+
+resource "random_string" "username" {
+  length  = 8
+  special = false
+  number  = false
+}
+
+resource "aws_security_group" "internal_tableau_db" {
+  vpc_id = "${var.apps_vpc_id}"
+
+  tags {
+    Name = "sg-${local.naming_suffix}"
+  }
+}
+
+resource "aws_security_group_rule" "allow_bastion" {
```

Comment:
Can the `aws_security_group_rule` resource rules be part of the `aws_security_group` please so that they are in line with the rest of the SGs in the VPCs?